### ISSUE TITLE
Contributing markdown doesn't contain information in regards to procedures and standards

### ISSUE THREAD

**Issue by nicholaspcr** [inspr/core-legacy#153]
**Is your feature request related to a problem? Please describe.**
The CONTRIBUTING markdown doesn't specify standards that should be followed when someone wants to contribute. For example, if I want to contribute by implementing a solution to an issue I don't have access to information that tells me standard procedures that should be followed. More specifically branch naming, if there is a commit format, what is the project layout etc. 

I might be mistaken but I believe the current content in the markdown could be merged with the `CODE_OF_CONDUCT` markdown, since it seems that the text was based of code_of_conduct files found in other repositories.

**Describe the solution you'd like**
A possible solution to this would be to create a documentation inspired by repositories like:
- https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md
- https://github.com/grafana/grafana/blob/HEAD/CONTRIBUTING.md
- https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md

**Additional context**
An example of table of contents that would be nice to have in a `CONTRIBUTION` markdown:
```
# Table of contents
- standard intro and contribution legal requirements
- Branching standards
  - naming
  - scope
  - PR and rebasing
- Commits
  - format
  - content
  
-  (Idea) maybe a reference to a documentation in regards to development?
```

Upon describing the table of contents of the CONTRIBUTION markdown the idea of separating the steps related to development on its own markdown came to me. Below is the example of what I think would also be a nice idea to complement the contribution markdown.

Table of contents `DEVELOPMENT.md`:
```
# Table of contents
- Development Environment
- Getting Started
   - Pre-requisites
   - Steps

- Using the CLI with the Development Environment
- Managing the Development Databases
   - Redis
- Project Structure
- Documentation
  - API - Definition of routes
  - Code Style
    - Code Formatting
    - Line Length
    - Formatting and Linting
  
- Testing
   - Unit Tests
   - End-to-end Tests
- Building and Running
- Releasing
- Troubleshooting
```

In short, the issue itself is about the lack of information about how to contribute
